File: lösungen_lurch/Python/ladebalken01.py
def print_loading_bar(percentage):
    full_loading_bar = 10

    print("[", end="")

    for i in range(1, percentage+1):
        print("#", end="")

    for i in range(percentage, full_loading_bar):
        print(" ", end="")

    print("]", end="")

    print(" " + str(percentage) + "0 %",  end="")
    print()


# user input
load = int(input("Enter a state:"))
if load >= 1 and load <= 10:
    print_loading_bar(load)
else:
    print("The number is too big or too low.")

File: lösungen_lurch/PHP/ladebalken01.php
<?php

function printLoadingBar($percentage) {
  $fullLoadingBar = 10;

  print "[";

  for ($i = 1; $i <= $percentage; $i++) {
    print "#";
  }

  for ($i = $percentage+1; $i <= $fullLoadingBar; $i++) {
    print " ";
  }

  print "]";

  print " " . $percentage . "0 %";
  print "\n";
}

// user input
$userInput = (int)readline("Enter a percentage:");
printLoadingBar($userInput);
